bat.d/66-wilco-ec

```diff
@@ -0,0 +1,730 @@
+        32)  cprintf "notice"  "* natacpi (%s) = inactive (disabled by configuration)\n" "$_batdrv_kmod" ;;
+        128) cprintf "err"     "* natacpi (%s) = inactive (no kernel support)\n" "$_batdrv_kmod" ;;
+        254) cprintf "warning" "* natacpi (%s) = inactive (laptop not supported)\n" "$_batdrv_kmod" ;;
+        *)   cprintf "err"     "* natacpi (%s) = unknown status\n" "$_batdrv_kmod" ;;
+    esac
+    if [ "$_bm_thresh" != "none" ]; then
+        printf "Parameter value ranges:\n"
+        printf "* START_CHARGE_THRESH_BAT0/1:  50..95(default)\n"
+        printf "* STOP_CHARGE_THRESH_BAT0/1:   55..100(default)\n"
+    fi
+    printf "\n"
+
+    # -- show battery data
+    local bat
+    local bcnt=0
+    local ed ef en
+    local efsum=0
+    local ensum=0
+
+    for bat in $_batteries; do # iterate batteries
+        batdrv_select_battery "$bat"
+
+        printf "+++ Battery Status: %s\n" "$bat"
+
+        printparm "%-59s = ##%s##" "$_bd_read/manufacturer"
+        printparm "%-59s = ##%s##" "$_bd_read/model_name"
+
+        print_battery_cycle_count "$_bd_read/cycle_count" "$(read_sysf "$_bd_read/cycle_count")"
+
+        if [ -f "$_bd_read/energy_full" ]; then
+            printparm "%-59s = ##%6d## [mWh]" "$_bd_read/energy_full_design" "" 000
+            printparm "%-59s = ##%6d## [mWh]" "$_bd_read/energy_full" "" 000
+            printparm "%-59s = ##%6d## [mWh]" "$_bd_read/energy_now" "" 000
+            printparm "%-59s = ##%6d## [mW]" "$_bd_read/power_now" "" 000
+
+            # store values for charge / capacity calculation below
+            ed=$(read_sysval "$_bd_read/energy_full_design")
+            ef=$(read_sysval "$_bd_read/energy_full")
+            en=$(read_sysval "$_bd_read/energy_now")
+            efsum=$((efsum + ef))
+            ensum=$((ensum + en))
+
+        elif [ -f "$_bd_read/charge_full" ]; then
+            printparm "%-59s = ##%6d## [mAh]" "$_bd_read/charge_full_design" "" 000
+            printparm "%-59s = ##%6d## [mAh]" "$_bd_read/charge_full" "" 000
+            printparm "%-59s = ##%6d## [mAh]" "$_bd_read/charge_now" "" 000
+            printparm "%-59s = ##%6d## [mA]" "$_bd_read/current_now" "" 000
+
+            # store values for charge / capacity calculation below
+            ed=$(read_sysval "$_bd_read/charge_full_design")
+            ef=$(read_sysval "$_bd_read/charge_full")
+            en=$(read_sysval "$_bd_read/charge_now")
+            efsum=$((efsum + ef))
+            ensum=$((ensum + en))
+
+        else
+            ed=0
+            ef=0
+            en=0
+        fi
+
+        print_batstate "$_bd_read/status"
+        printf "\n"
+
+        if [ "$verbose" -eq 1 ]; then
+            printparm "%-59s = ##%6s## [mV]" "$_bd_read/voltage_min_design" "" 000
+            printparm "%-59s = ##%6s## [mV]" "$_bd_read/voltage_now" "" 000
+            printf "\n"
+        fi
+
+        # --- show battery features: thresholds
```

Comment:
Because the thresholds are battery independent this output block should be moved out of the bat loop resp. the _+++ Battery Status status_ section and attached to the preceding _+++ Battery Care_ section.